site/docs/v1/tech/themes/index.adoc

```diff
@@ -90,6 +115,11 @@ This page is rendered when a user clicks the URL from the verification email and
 +
 [uri]#/email/verify#
 
+[field]#Index# [required]#Required# [since]#Since 1.26.0#::
+This is the root landing page for FusionAuth.
```

Comment:
I'd add a bit more here:

"This is the root landing page. This page is available to unauthenticated users and will be displayed whenever someone navigates to the FusionAuth host's root page."